src/shared/codelens/localLambdaRunner.ts

```diff
@@ -515,16 +484,27 @@ export const invokeLambdaFunction = async (params: {
         templatePath: params.samTemplatePath,
         eventPath,
         environmentVariablePath,
-        debugPort: (params.isDebug && params.debugConfig) ? params.debugConfig.port.toString() : undefined,
-        invoker: params.samTaskInvoker
+        invoker: params.samTaskInvoker,
+        debugPort: (params.isDebug && params.debugPort) ? params.debugPort.toString() : undefined,
+        debuggerPath: params.debuggerPath
     })
 
     const startInvokeTime = new Date()
     await command.execute()
 
-    if (params.isDebug && params.debugConfig) {
+    if (params.isDebug) {
```

Comment:
Can we swap this for a check on `debugConfig` and `debugPort` and eliminate the need for the `isDebug` param?